en/contribute/README.md

```diff
@@ -112,6 +112,24 @@ PX4 uses [astyle](http://astyle.sourceforge.net/) for code formatting. Valid ver
 
 Once installed, formatting can be checked with `./Tools/astyle/check_code_style_all.sh`. The output should be `Format checks passed` on a clean master. If that worked, `make format` can be used in the future to check and format all files automatically.
 
+### In-Source Documentation
```

Comment:
This bit is what needs review.

Reply:
I apologize that I am late on this, but this all looks great.  Tested on a new repo clone:
![image](https://user-images.githubusercontent.com/2497951/44928120-10f14a80-ad14-11e8-8ff2-57007489d4d5.png)

Reply:
Thanks for your work here @hamishwillee

Reply:
Thanks @mcsauder for review. Now we all just need to start gently suggesting this when we see docs that could use some additional information.